docs/product-manuals/cloud-console/manage-clusters/create-cluster.md

```diff
@@ -3,8 +3,10 @@ id: create-cluster
 title: Create a cluster
 ---
 
-If you haven't created a cluster yet, the overview will be empty.
+If you haven't created a cluster yet, the **Clusters** page will be empty.
 
 import CreateCluster from './create-cluster-include.md'
 
 <CreateCluster/>
+
+[//]:# (Seems as though this is the method of calling something like a React component into a Markdown file. Should this have more instruction or should I directly link the file in place of the lines above?)
```

Comment:
@christinaausley The snippet above is used to include another Markdown file into this one. On this page everything from the `create-cluster-include.md` file will be included. So nothing else to do here :)

Reply:
Great, thanks for confirming!